### DISCUSSION TITLE
Add "manage_runners:org" permission

### DISCUSSION THREAD

**Discussion by fsamier** [community/community#17260] [Actions] [answered]
The only way we've found to share a runner across several repositories is to declare it at organisation level. 

However, the developers are managing the runners, but according to [documentation](https://docs.github.com/en/rest/actions/self-hosted-runners#create-a-registration-token-for-an-organization) in order to generate a registration token, the PAT must have the `admin:org` permission, which is far too broad for this task. We don't want to promote all developers as owner, and we are not confident having a such powerful token being exchanged.
As this permission granularity exists at entreprise level, it would make sense to add it at organisation level.

Another acceptable solution would be to allow generating a registration token for several repositories at once, as most developers are admin on repositories they manage, their own token could be used as they have the required `repo` permissions.

**Comment by debugger24** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/12707743/184866388-d620912d-ce67-4fc8-88e9-1c740c7817c6.png)

Looks like they have added it now.

Tested it, it didn't work. 

```
"failed to create registration token: POST https://api.github.com/orgs/demo-gh-app/actions/runners/registration-token: 403 Must have admin rights to Repository. []"}
```

**Reply by fsamier** [reply]
It is part of the roadmap now: https://github.com/github/roadmap/issues/503
Not yet shipped though
